CMakeLists.txt

```diff
@@ -102,22 +117,28 @@ macro(jerry_add_link_flags)
 endmacro()
 
 # build mode specific compile/link flags
+if(USE_GCC)
 set(CMAKE_C_FLAGS_RELEASE "-Os")
+endif()
 
 # Architecture-specific compile/link flags
 jerry_add_compile_flags(${FLAGS_COMMON_ARCH})
 jerry_add_flags(CMAKE_EXE_LINKER_FLAGS ${FLAGS_COMMON_ARCH})
 
 # Enable static build
 if(ENABLE_STATIC_LINK)
-  jerry_add_link_flags("-static")
+  if (USING_GCC_OR_CLANG)
+    jerry_add_link_flags("-static")
+  endif()
 endif()
 
 # LTO
 if(ENABLE_LTO)
-  jerry_add_compile_flags(-flto)
-  jerry_add_link_flags(-flto)
-  if(CMAKE_COMPILER_IS_GNUCC)
+  if (USING_GCC_OR_CLANG)
```

Comment:
You're right -- I missed this one...  The TI compiler flag is slightly different, and I've added it, here.